test/batch_norm.jl

```diff
@@ -0,0 +1,26 @@
+using Test
+using Bijectors, ForwardDiff, LinearAlgebra
+using Random: seed!
+
+seed!(1)
+
+@testset "InvertibleBatchNorm" begin
+    z = randn(20, 2)
```

Comment:
Is this supposed to be a batch of 20 2-dim inputs? If so, this should be `(2, 20)`. Distributions.jl use this kind of shape  (because iterating through [:, i] is faster than [i, :]) so also follow the same convention.

Reply:
Yes we should stick to column major for sure as it's more efficient in Julia. The original implementaion from Flux is column major so it looks like a mistake in the test for me. @sharanry can you give a try to fix?